docs/blog/2023-community-kickoff.rst

```diff
@@ -0,0 +1,122 @@
+.. post:: Jan 17, 2023
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Kickoff - January 2023
+===============================================
+
+Hey folks, Eric Holscher here, one of the co-founders of Write the Docs.
+I'm here with the traditional January community update,
+which goes out in place of the newsletter. 
+
+We're excited about a lot of things this year,
+but most of all we're excited to see you all again in-person.
+
+Our 2023 conferences
+--------------------
+
+Thanks to everyone who attended our virtual events in 2022.
+We had wonderful attendance in Portland, Prague, and Australia.
+We continue to appreciate the warmth and spirit that you bring to our virtual events,
+and look forward to more of it in 2023.
+
+We are excited this year for a partial return to in-person events,
+and a new experiment for our virtual event:
+
+* `Portland <https://www.writethedocs.org/conf/portland/2023/>`_ will be in-person on **May 7-9**.
+* `Atlantic <https://www.writethedocs.org/conf/atlantic/2023/>`_ will be a virtual event, spanning the US and European timezones, on **September 10-12**.
+
+We hope that you'll work with us to make all of our events unique again this year by submitting talks,
+and showing up with your usual enthusiasm and warmth.
+
+Team restructuring
+------------------
+
+There are some changes coming to our teams this year,
+specifically the `CoreOps team <https://www.writethedocs.org/team/#core-operations-coreops>`_ and new `Community board <https://www.writethedocs.org/team/#community-board>`_.
+
+The *CoreOps team* is a small group of us that have been working to make Write the Docs happen for a long time.
+Historically this group has included our Portland & Prague conference chairs,
+along with a couple other folks heavily involved in the overall operation of Write the Docs.
+It's responsible for the tasks involved in keeping Write the Docs a viable organization.
+
+The major change happening there is that Mikey Ariel will moving into a new role focused on non-conference community development.
+Sasha Romijn will be filling the role of conference chair for our Atlantic conference.
+The other members of the CoreOps team, myself & Samuel Wright, will continue with our current roles.
+
+The *Community board* is new, and tasked with the overall direction of the community.
+The members are the team leads of each of the Write the Docs team,
+as well as the conference chairs of our various conferences.
+Our goal is to have meetings quarterly where we all share the status of what we're working on,
+and any help that we could use in keeping the various teams functioning effectively.
+We're hoping that having more defined communication will help people feel more connected and supported in the work we're all doing.
+
+You can see the full set of changes in `pull request 1811 <https://github.com/writethedocs/www/pull/1811>`_.
```

Comment:
Maybe it makes sense to link to the WEP rather than the website change PR? The WEP has the motivation etc...

Reply:
:+1:

Reply:
I included that text in the PR description, otherwise the WEP is a bit light and points back to the PR for details :)

Reply:
Updated anyway.